web-app/js/portal/utils/ObservableUtils.js

```diff
@@ -0,0 +1,16 @@
+Ext.namespace('Portal.utils');
+
+Portal.utils.ObservableUtils = {
+    FUNCTIONS_TO_FORWARD: ['on', 'un', 'fireEvent'],
```

Comment:
Thinking a bit more I don't think I'd like to see the events being configurable when choose to make an object Observable with this class. As Observable is an interface we should really guarantee that it implements the functions which all Observable objects should have.

So I think I'll update this list to be complete, rather than just the ones I used.